app/controllers/project_database.py

```diff
@@ -166,7 +166,38 @@ def get(self, project_id):
         if errors:
             return dict(status='fail', message=errors), 500
 
-        return dict(status='success', data=dict(databases=json.loads(database_data))), 200
+        database_data_list = json.loads(database_data)
+
+        # Check the database status on host
+        for database in database_data_list:
+            flavour_name = database['database_flavour_name']
+            if not flavour_name:
+                flavour_name = "mysql"
+
+            db_flavour = get_db_flavour(flavour_name)
+            database_service = db_flavour['class']
+
+            try:
+                database_connection = database_service.create_db_connection(
```

Comment:
Hey @MutegekiHenry , Great work.
But i added a function in db_service helper class called `check_db_connection` that returns a `false` or `true`. this does not require the closing of connections from your end. I suggest that you utilise it, instead of `create_db_connection`

Reply:
@Mubangizi thanks for the feedback. However I have checked out the `check_db_connection` function and it checks the connection based on the super user credentials yet this ticket is meant to check the individual status of each db. I noticed that some dbs cant actually connect due to wrong credentials while testing this end point which we might never get to tell using `check_db_connection` function.

Reply:
True